package com.example.javademo.throwable;

public class ExceptionDemo3 {
    public static void main(String[] args) throws ArithmeticException{

        //运行时异常
        int a = 5;
        int b = 0;
        System.out.println(a/b);

        System.out.println("hello");
    }
}
